pkgs/servers/web-apps/pump.io/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitHub, makeWrapper, callPackage, nodejs, python, utillinux, graphicsmagick }:
+
+with stdenv.lib;
+
+let
+  nodePackages = callPackage (import ../../../top-level/node-packages.nix) {
+    inherit stdenv nodejs fetchurl fetchgit;
+    neededNatives = [ python ] ++ optional stdenv.isLinux utillinux;
+    self = nodePackages;
+    generated = ./package.nix;
```

Comment:
@rvl i assume you generated this package.nix with npm2nix, right?

if yes, maybe it would be better to create equivalent node-packages.json, like it did for statsd (https://github.com/NixOS/nixpkgs/tree/master/pkgs/tools/networking/statsd). i think it would be easier to update a package later on this way, but that is just my opinion.